docs/data-ai/data/query.md

```diff
@@ -425,6 +425,31 @@ For information on connecting to your Atlas instance from other MQL clients, see
 
 On top of querying sensor data with third-party tools, you can also [query it with the Python SDK](/data-ai/reference/data-client/) or [visualize it](/data-ai/data/visualize/).
 
+## Query optimization and performance best practices
+
+1. When querying large datasets, whether from default storage or a [hot data store](/data-ai/data/hot-data-store/), you can improve the query's efficiency by specifying the following parameters in the query:
+
+   - `organization_id`
+   - `location_id`
+   - `machine_id`
+   - `part_id`
+   - `component_type`
+   - `component_name`
+   - `method_name`
+   - `date`
```

Comment:
`date` is not a valid query parameter. But I stand corrected from my earlier slack comment. Seems that you **can** query by `capture_date`. You could then get even more specific and query by something like `time_received`